### PR TITLE
Added Footer

### PR DESCRIPTION
**Pull request by kongsally**

### CODE REVIEW

**Comment by kongsally** [comment]
r @nancywong @thomaslee1911

File: app/assets/styles/app.scss [hunk 1]
```diff
@@ -122,3 +126,29 @@ p, label, input {
...
+	bottom: 0;
+	width: 100%;
+	text-align: right;
+	padding: 1em;
+	margin-top: 5%;
+	background-color: $dark-teal;
+}
+
+#h4i_logo {
+	content: url("../../static/images/h4i_square_white.png");
```
Comment (rileynwong) [inline app/assets/styles/app.scss:140]: If the logos are used more than once, would consider setting the png urls as a sass variables

File: app/static/styles/app.css [hunk 2]
```diff
@@ -4,6 +4,11 @@ nav {
 }
 
 @media -sass-debug-info{filename{}line{font-family:\0000318}}
+.body.container {
```
Comment (rileynwong) [inline app/static/styles/app.css:7]: Maybe add `*.css` to gitignore since we only look at sass files

Comment (sandlerben) [inline app/static/styles/app.css:7]: No don't do this. Some libraries we depend on use css files. Also, not everyone who clones our code should need a sass compiler installed to run the app.

File: app/templates/partials/_footer.html [hunk 3]
```diff
@@ -0,0 +1,6 @@
+<footer>
+    <div>
+    	<h3>Built with <i class="heart icon"></i> by </h3>
+    	<h3> <img id = "h4i_logo">Hack4Impact</h3>
+    </div>
+</footer>
```
Comment (sandlerben) [inline app/templates/partials/_footer.html]: Add a newline to the end of your files please

File: app/templates/layouts/base.html [hunk 4]
```diff
@@ -5,20 +5,23 @@
...
+            {# Implement CSRF protection for site #}
+            {% if csrf_token() %}
+                <div style="visibility: hidden; display: none">
+                    <input type="hidden" name="csrf_token" value="{{ csrf_token() }}">
+                </div>
+            {% endif %}
+        </div>
+        {% include 'partials/_footer.html' %}
     </body>
 </html>
```
Comment (sandlerben) [inline app/templates/layouts/base.html]: See below

**Comment by sandlerben** [comment]
lgtm

**Pull request merged**